ui-v2/src/components/ui/flow-run-acitivity-bar-graph/flow-run-activity-bar-graph.test.tsx

```diff
@@ -62,12 +67,24 @@ describe("FlowRunActivityBarChart", () => {
 	});
 
 	it("shows tooltip on bar hover", () => {
-		render(
-			<RouterContextProvider router={router}>
-				{/* @ts-expect-error - Type error from test data not matching schema */}
+		const rootRoute = createRootRoute({
+			component: () => (
+				// @ts-expect-error - Type error from test data not matching schema
 				<FlowRunActivityBarChart {...defaultProps} numberOfBars={1} />
-			</RouterContextProvider>,
-		);
+			),
+		});
+		const router = createRouter({
+			routeTree: rootRoute,
+			history: createMemoryHistory({
+				initialEntries: ["/"],
+			}),
+			context: {
+				queryClient: new QueryClient(),
+			},
+		});
+
+		// @ts-expect-error - Type error from using a test router
+		render(<RouterProvider router={router} />);
```

Comment:
This appears to be the 'best' way to test components with a router dependency based on the [`@tanstack/react-router` tests](https://github.com/TanStack/router/blob/main/packages/react-router/tests/routeContext.test.tsx).

Reply:
We can probably add this to [our test utils](https://github.com/PrefectHQ/prefect/blob/main/ui-v2/tests/utils/index.ts) to expedite the router wrapping for future tests